contracts/lib/platform/TokenDotFactory.sol

```diff
@@ -134,14 +136,13 @@ contract TokenDotFactory is Ownable {
     }
 
     //https://ethereum.stackexchange.com/questions/15350/how-to-convert-an-bytes-to-address-in-solidity
-    function bytesToAddr (bytes b) public pure returns (address) {
-        uint result = 0;
-        for (uint i = b.length-1; i+1 > 0; i--) {
-            uint c = uint(b[i]);
-            uint to_inc = c * ( 16 ** ((b.length - i-1) * 2));
-            result += to_inc;
-        }
-        return address(result);
+    function bytesToAddr (bytes memory bys) public pure returns (address) {
```

Comment:
old bytesToAddr function threw error at line 140  uint c = uint(b[i]);

could not cast bytes to uint


my work around was to find another algorithm online to caste bytes to address